core/include/scipp/core/view_index.h

```diff
@@ -93,9 +93,9 @@ class SCIPP_CORE_EXPORT ViewIndex {
   // loops are used.
 
   scipp::index m_index{0};
-  std::array<scipp::index, NDIM_MAX> m_delta = {0, 0, 0, 0, 0, 0};
-  std::array<scipp::index, NDIM_MAX> m_coord = {0, 0, 0, 0, 0, 0};
-  std::array<scipp::index, NDIM_MAX> m_extent = {0, 0, 0, 0, 0, 0};
+  std::array<scipp::index, NDIM_MAX> m_delta = {};
+  std::array<scipp::index, NDIM_MAX> m_coord = {};
+  std::array<scipp::index, NDIM_MAX> m_extent = {};
```

Comment:
:+1: I did not know you can do this.

Reply:
Seems to be completely supported as part of ```value initialisation```